### PR TITLE
Updates access code text in line with style guide

### PR DESCRIPTION
**Pull request by balibirchlee**
### What is the context of this PR?

Updates to Access Code text in line with style guide, around links, contractions and specificity. fixes: https://github.com/ONSdigital/design-system/issues/2959

### How to review this PR

Run locally and open out pages for access code & access code error - check text.

### Checklist

This needs to be completed by the person raising the PR.

- [x] I have selected the correct Assignee
- [x] I have linked the correct Issue

### CODE REVIEW

File: src/components/access-code/example-access-code-error.njk [hunk 1]
```diff
@@ -58,18 +58,19 @@ layout: ~
         "classes": "ons-u-mb-xl"
     }) }}
 
-    <h2>If you don’t have an access code</h2>
+    <h2>If you do not have an access code</h2>
     <p>
-        You can <a href="#0">request a new access code</a> to start a new study if you have lost or not received an access code. This can be sent to you by text or post.
+        If you have lost or not received an access code, you can request a new access code <a href="#0">[LINK]</a>.
```
Comment (rmccar) [inline src/components/access-code/example-access-code-error.njk:63]:
```suggestion
        If you have lost or not received an access code, you can <a href="#0">request a new access code</a>.
```

File: src/components/access-code/example-access-code-error.njk [hunk 2]
```diff
@@ -58,18 +58,19 @@ layout: ~
...
+        This can be sent to you by text or post.
     </p>
 
     {% set content %}
-        <p>If you need to answer separately from the people you live with, you can <a href='#0'>request an individual access code</a> to start a separate study.</p>
+        <p>If you need to answer separately from the people you live with, you can request an individual access code <a href="#0">[LINK]</a>.</p>
```
Comment (rmccar) [inline src/components/access-code/example-access-code-error.njk:68]:
```suggestion
        <p>If you need to answer separately from the people you live with, you can <a href="#0">request an individual access code</a>.</p>
```

File: src/components/access-code/example-access-code.njk [hunk 3]
```diff
@@ -34,18 +34,19 @@ layout: ~
         "classes": "ons-u-mb-xl"
     }) }}
 
-    <h2>If you don’t have an access code</h2>
+    <h2>If you do not have an access code</h2>
     <p>
-        You can <a href="#0">request a new access code</a> to start a new study if you have lost or not received an access code. This can be sent to you by text or post.
+        If you have lost or not received an access code, you can request a new access code <a href="#0">[LINK]</a>.
```
Comment (rmccar) [inline src/components/access-code/example-access-code.njk:39]:
```suggestion
        If you have lost or not received an access code, you can <a href="#0">request a new access code</a>.
```

File: src/components/access-code/example-access-code.njk [hunk 4]
```diff
@@ -34,18 +34,19 @@ layout: ~
...
+        This can be sent to you by text or post.
     </p>
 
     {% set content %}
-        <p>If you need to answer separately from the people you live with, you can <a href='#0'>request an individual access code</a> to start a separate study.</p>
+        <p>If you need to answer separately from the people you live with, you can request an individual access code <a href="#0">[LINK]</a>.</p>
```
Comment (rmccar) [inline src/components/access-code/example-access-code.njk:44]:
```suggestion
        <p>If you need to answer separately from the people you live with, you can <a href="#0">request an individual access code</a>.</p>
```

**Review by rmccar — CHANGES_REQUESTED** [review]
Moved the link text inside the link. Also the button can be removed from the details components, this seems to be left behind from the previous implementation. The button param has now been removed.

File: src/components/access-code/example-access-code-error.njk [hunk 5]
```diff
@@ -58,30 +58,21 @@ layout: ~
...
-            "title": "Need to answer separately from your household?",
-            "button": {
-                "close": "Hide this",
-                "contextSuffix": "content"
-            }
-        })
-    %}
-        {{ content | safe }}
-    {% endcall %}
+
```
Comment (rmccar) [inline src/components/access-code/example-access-code-error.njk:77]: We don't want to remove the details component completely, just the button. Same with the other example

```suggestion
{% call onsDetails({
            "id": "details-access-code-error",
            "title": "Need to answer separately from your household?"
        })
    %}
        {{ content | safe }}
    {% endcall %}
```

**Review by rmccar — APPROVED** [review]

**Review by alessioventuriniAND — APPROVED** [review]

**Comment by alessioventuriniAND** [comment]
replaced by #2993

**Pull request closed without merging**